### PR TITLE
Test MME hosting with MultiDataModel

### PR DESCRIPTION
**Pull request by ajaykarpur**
Adds testing for multi-model endpoint hosting.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/1.4.0/final/Dockerfile.gpu [hunk 1]
```diff
@@ -3,6 +3,7 @@ ARG py_version
 FROM awsdeeplearningteam/mxnet-model-server:base-gpu-py$py_version
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```
Comment (ChuyangDeng) [inline docker/1.4.0/final/Dockerfile.gpu]: Multi-model mode is only supported in CPU containers

File: docker/1.4.1/py2/Dockerfile.gpu [hunk 2]
```diff
@@ -1,6 +1,7 @@
 FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu16.04
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```
Comment (ChuyangDeng) [inline docker/1.4.1/py2/Dockerfile.gpu]: Same as above

File: docker/1.4.1/py3/Dockerfile.gpu [hunk 3]
```diff
@@ -3,6 +3,7 @@ FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu16.04
 LABEL maintainer="Amazon AI"
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```
Comment (ChuyangDeng) [inline docker/1.4.1/py3/Dockerfile.gpu]: Same as above

File: docker/1.6.0/py2/Dockerfile.gpu [hunk 4]
```diff
@@ -5,6 +5,9 @@ LABEL maintainer="Amazon AI"
 # Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
 # https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```
Comment (ChuyangDeng) [inline docker/1.6.0/py2/Dockerfile.gpu]: remove MME label from GPU images

File: docker/1.6.0/py3/Dockerfile.gpu [hunk 5]
```diff
@@ -5,6 +5,9 @@ LABEL maintainer="Amazon AI"
 # Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
 # https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```
Comment (ChuyangDeng) [inline docker/1.6.0/py3/Dockerfile.gpu]: remove MME label from GPU images

File: docker/1.4.0/final/Dockerfile.gpu (hunk 1 above)
Comment (ajaykarpur) [reply docker/1.4.0/final/Dockerfile.gpu]: Removed

File: docker/1.4.1/py3/Dockerfile.gpu (hunk 3 above)
Comment (ajaykarpur) [reply docker/1.4.1/py3/Dockerfile.gpu]: Removed

File: docker/1.6.0/py3/Dockerfile.gpu (hunk 5 above)
Comment (ajaykarpur) [reply docker/1.6.0/py3/Dockerfile.gpu]: Removed

File: docker/1.4.1/py2/Dockerfile.gpu (hunk 2 above)
Comment (ajaykarpur) [reply docker/1.4.1/py2/Dockerfile.gpu]: Removed

File: docker/1.6.0/py2/Dockerfile.gpu (hunk 4 above)
Comment (ajaykarpur) [reply docker/1.6.0/py2/Dockerfile.gpu]: Removed

**Pull request merged**